openc3/lib/openc3/models/offline_access_model.rb

```diff
@@ -19,28 +19,29 @@
 require 'openc3/models/model'
 
 module OpenC3
+  # Note: This model is locked to the DEFAULT scope
```

Comment:
Is this because you really only need 1 offline access token and the DEFAULT scope is always there?

Reply:
Exactly.